File: frontend/src/app/models/todo.ts
export interface Todo {
  id: string;
  title: string;
  description: string;
  isCompleted: boolean;
  createdAt: string;
}

export interface CreateTodoRequest {
  title: string;
  description: string;
  isCompleted: boolean;
}

export interface UpdateTodoRequest {
  title: string;
  description: string;
  isCompleted: boolean;
}

File: backend/ToDo.WebAPI/Todoproject/Entities/Entity/Todo.cs
namespace ToDo.WebAPI.Todoproject.Entities.Entity;

public class Todo
{
    public Guid Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public bool IsCompleted { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    public string UserId { get; set; } = string.Empty;
}